internal/martian/proxy_connect.go

```diff
@@ -122,5 +121,19 @@ func (p *Proxy) connectSOCKS5(req *http.Request, proxyURL *url.URL) (*http.Respo
 		return nil, nil, err
 	}
 
-	return proxyutil.NewResponse(200, http.NoBody, req), conn, nil
+	return newConnectResponse(req), conn, nil
+}
+
+func newConnectResponse(req *http.Request) *http.Response {
+	return &http.Response{
+		StatusCode: http.StatusOK,
```

Comment:
Please set `Status` too. The response log lacks it, you can see it in commit message.
```
sc-1         | HTTP/2.0
```
vs 
```
sc-server-1  | HTTP/1.1 200 OK
```

Reply:
Done and donner

Reply:
Wrong 🤔. That's how it's done in proxyutil 
```
Status:     fmt.Sprintf("%d %s", code, http.StatusText(code)),
```

Reply:
That is the best way of doing it, fixed.